client/root.js

```diff
@@ -27,7 +27,7 @@ function setupLoggedOut() {
 	if ( config.isEnabled( 'desktop' ) ) {
 		page( '/', () => {
 			if ( config.isEnabled( 'oauth' ) ) {
-				page.redirect( '/authorize' );
+				page.redirect( '/oauth-login' );
```

Comment:
I see another reference to `oauth-login` in [`client/auth/index.js`](https://github.com/Automattic/wp-calypso/blob/f536fe8fb05e438031fb7471c277a9d5f8cde7c8/client/auth/index.js#L16):

`page( '/oauth-login', controller.oauthLogin, makeLayout, clientRender );`

Do we need to add the `controller.oauthLogin`,  `makeLayout` and  `clientRender` arguments here as well?

Reply:
That reference defines the handlers for the `/oauth-login` route. These handles render the OAuth login React UI.

The `page.redirect` call is something different -- it redirects the browser to the `/oauth-login` route.